### DISCUSSION TITLE
Internal email only

### DISCUSSION THREAD

**Discussion by kennylajara** [rspamd/rspamd#3774] [Ideas]
I want to have a `sensitive.example.com` subdomain for sensitive emails. My idea is to block any outgoing email from (example) `reports@sensitive.example.com` if they are sent to any external email (like Gmail or Outlook) but I don't want to block it if is sent from a `@sensitive.example.com` account to `name@example.com` or `name@sales.example.com`, for example.

Is it possible to create a rule like that?

**Comment by moisseev** [comment]
local.d/multimap.conf
```cxx
SENSITIVE_FROM_DOMAIN {
  type = "combined";
  rules {
    from {
      map = "$LOCAL_CONFDIR/local.d/multimap.d/sensitive-from.map";
      selector = "from:domain";
    }
    rcpt {
      map = "$LOCAL_CONFDIR/local.d/multimap.d/sensitive-rcpt.map";
      selector = "rcpts:addr";
    }
  }
  expression = "from & !rcpt"
  action = "reject";
}
```

local.d/multimap.d/sensitive-from.map
```
sensitive.example.com
```

local.d/multimap.d/sensitive-rcpt.map
```
name@example.com
name@sales.example.com
```

**Reply by kennylajara** [reply]
Hi, Thank. 

I made the config and is not working. I guess the system is not verifying outgoing emails at all... Where do I check that?

Can I just list the domain in `local.d/multimap.d/sensitive-rcpt.map` so I don't have to add each email, each time? Better yet, Can I use some kind of regex to whitelist `example.com` and `*.example.com` (any subdomain of `example.com`)?

**Reply by moisseev** [reply]
> I guess the system is not verifying outgoing emails at all...

If you are using postfix and sendiing mail locally you should see this:
https://rspamd.com/doc/tutorials/scanning_outbound.html#mta-with-milter-support-eg-postfix-or-sendmail

> Where do I check that?

In the `rspamd.log`.

> Can I just list the domain in `local.d/multimap.d/sensitive-rcpt.map` so I don't have to add each email, each time? Better yet, Can I use some kind of regex to whitelist `example.com` and `*.example.com` (any subdomain of `example.com`)?

Yes, you can list domains or regexps in a map. Please see multimap module documentation for details.
